dayone2_json_to_markdowns/cli.py

```diff
@@ -58,7 +58,10 @@ def _process_entry(self, entry):
         utc_timestamp = dateutil.parser.parse(creation_date)
         local_timestamp = utc_timestamp.astimezone(self._entry_tzinfo(entry))
 
-        text = entry["text"]
+        try: 
+            text = entry["text"]
+        except: 
+            text = u""
```

Comment:
A good point! 👍 

How about doing like this?

```python
text = entry.get("text", u"")
```

Reply:
Ah, yes, that is much better. I'll update the PR.